frontend/src/pages/AuthPage/index.jsx

```diff
@@ -28,9 +28,15 @@ const AuthPage = ({ uid, setUid }) => {
     }
     history.push("/eidt_profile/" + uid);
```

Comment:
Do you refer to "edit_profile" for the path?

Reply:
Yes, I will fix it